### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by Mukaba**
## Details popup window

**Description:** This pull request updates the previous merged PRs by adding an interactive popup window to project cards. 
This upgrade includes the following:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- For each project, I've added the object name, description, featured image, technologies, link to the live version, and link to the source.
- The main page is created dynamically using the information stored in that JavaScript object.

_I collaborated with my coding partner_ 

Great regards

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Mukaba, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no commits from your partner. Kindly make sure your partner has commits on your `popups` branch

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: assets/style/style.css [hunk 1]
```diff
@@ -407,41 +425,92 @@ body {
...
-.tonic2__header div {
-  background: no-repeat center url(../img/card_tonic2.svg);
+.project__popup {
+  width: 90%;
+  height: 90%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  overflow: scroll;
+  padding: 1rem;
```
Comment (zieeco) [inline assets/style/style.css:479]: - Please, let's use the correct color for the paragraph in the popup and in the `ABOUT SECTION` to make the design close to the Figma as much as we can. See image below:

*Figma design*
Screenshot from 2023-03-08 14-06-48

*Your design*
Screenshot from 2023-03-08 14-05-21

*Figma design*
Screenshot from 2023-03-08 14-23-05

*Your design*
Screenshot from 2023-03-08 14-22-52

File: assets/style/style.css [hunk 2]
```diff
@@ -407,41 +425,92 @@ body {
   gap: 15px;
 }
```
Comment (zieeco) [inline assets/style/style.css:430]: - Please, let's improve the design on each card by improving the image height to show correctly as seen in the Figma design
- Please, let's use the correct color for the paragraph content in each card to make the design close to the Figma as much as we can. See image below:



*Figma design*
Screenshot from 2023-03-08 14-17-44

*Your design*
Screenshot from 2023-03-08 14-16-59

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit message :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Mukaba 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

No linter errors.✔️
Nice design. ✔️
Descriptive PR. ✔️
Good commit message ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/style/style.css (hunk 2 above)
Comment (Mukaba) [reply assets/style/style.css:430]: Thanks for your feedback dear @zieeco

**Pull request merged**